tools/topology/topology2/production/tplg-targets-ace1.cmake

```diff
@@ -55,6 +55,13 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-rt712-l0-4ch.bin,\
 HDMI1_ID=5,HDMI2_ID=6,HDMI3_ID=7,DMIC0_ENHANCED_CAPTURE=true,\
 EFX_DMIC0_TDFB_PARAMS=line4_pass,EFX_DMIC0_DRC_PARAMS=dmic_default"
 
+"cavs-sdw\;sof-mtl-tas2783-2ch\;PLATFORM=mtl,NUM_SDW_AMP_LINKS=1,\
+SDW_JACK=false,NUM_DMICS=2,\
```

Comment:
I think you are missing these:
```
,PDM1_MIC_A_ENABLE=0,PDM1_MIC_B_ENABLE=0,DMIC0_ID=5,DMIC1_ID=6,\
```
Without it the default build will create 4ch DMIC blob, the IDs needs to be specified (thanks @bardliao  for the numbers!)